ui/components/multichain/nft-item/nft-item.js

```diff
@@ -85,15 +96,26 @@ export const NftItem = ({
                 clickable={clickable}
               />
             ) : (
-              <Box
-                className="nft-item__item nft-item__item-image"
-                data-testid="nft-image"
-                as="img"
-                src={src}
-                alt={alt}
-                display={Display.Block}
-                justifyContent={JustifyContent.center}
-              />
+              <>
+                {src ? (
+                  <Box
+                    className="nft-item__item nft-item__item-image"
+                    data-testid="nft-image"
+                    as="img"
+                    src={src}
+                    alt={alt}
+                    display={Display.Block}
+                    justifyContent={JustifyContent.center}
+                  />
+                ) : (
+                  <NftDefaultImage
+                    className="nft-item__default-image"
+                    data-testid="nft-default-image"
+                    name={name}
+                    tokenId={tokenId}
+                  />
+                )}
+              </>
```

Comment:
This block of code seems to be duplicated with the one above. Can we extract to a separate `renderERC155Image` function to reduce duplicity?